dev/io.openliberty.jakarta.batch.2.0/.externalToolBuilders/org.eclipse.pde.ManifestBuilder (3).launch

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?><launchConfiguration type="org.eclipse.ant.AntBuilderLaunchConfigurationType">
```

Comment:
I suspect the file name "ManifestBuilder (3).launch" is a mistake.  If so, please re-add the file with the correct name.